### PR TITLE
Added JNI Support and PasswordManager.

### PR DESCRIPTION
**Pull request by rip32700**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -27,20 +27,13 @@
       </value>
     </option>
   </component>
-  <component name="ProjectLevelVcsManager" settingsEditedManually="false">
-    <OptionsSetting value="true" id="Add" />
```
Comment (maratsubkhankulov) [inline .idea/misc.xml]: I don't think we want to track this file in git

File: app/src/main/java/com/wallet/crypto/trust/controller/PasswordManager.java [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+/**
+ * Password Manager that should assist
+ * in securely handling the user's password.
+ *
+ * Created by Philipp Rieger on 21.10.17.
+ */
+
+public final class PasswordManager {
+
+    public static final String MyPREFERENCES = "TrustWalletPrefs" ;
```
Comment (maratsubkhankulov) [inline app/src/main/java/com/wallet/crypto/trust/controller/PasswordManager.java]: Note that we're already using shared preferences e.g. in SettingsFragment.java

**Comment by maratsubkhankulov** [comment]
This looks good. I'd like to ask for an extension - we need the ability to store passwords for multiple accounts and associate those passwords with the account address.

**Comment by rip32700** [comment]
Check again, if sufficient now

**Pull request merged**